proto/elys/stablestake/debt.proto

```diff
@@ -26,4 +26,27 @@ message Debt {
   uint64 borrow_time = 5;
   uint64 last_interest_calc_time = 6;
   uint64 last_interest_calc_block = 7;
+  uint64 pool_id = 8;
 }
+
+message LegacyDebt {
```

Comment:
SInce there is just the addition of a new field don't think we need to have Legacy Debt, Protobuf allows that

Reply:
I am not sure how it will work, we need to first get old values whose type is different due to addition of fields

this method is tested and working fine

Reply:
updated